scenarios/kubernetes_e2e.py

```diff
@@ -442,6 +459,9 @@ def create_parser():
         '--test', default='true', help='If we need to set --test in e2e.go')
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
+    parser.add_argument(
+        '--multiple-federations', default=False, action='store_true',
```

Comment:
How would that help? The point is to get the node name and the executor number here to give a unique name from the pool of available names for each instance of the run. I can move it to `kubetest`, but where would I get that information from in `kubetest`? Node name and the executor number are not injected into the environment by the time you get to `kubetest.`